tests/pipeline/test_pipeline_load_info.py

```diff
@@ -0,0 +1,26 @@
+import dlt
+
+
+data = [
+    {"id": 1, "name": "Alice"},
+    {"id": 2, "name": "Bob"},
+]
+
+
+def test_pipeline_load_info_metrics_schema_is_not_chaning() -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name="quick_start",
+        destination="duckdb",
+        dataset_name="mydata",
+    )
+
+    load_info = pipeline.run(data, table_name="users")
+
+    pipeline.run([load_info], table_name="_load_info")
+    first_version_hash = pipeline.default_schema.version_hash
+
+    load_info = pipeline.run(data, table_name="users")
```

Comment:
here let's load again but we should add another source with different resource and some schema hints